pyanaconda/modules/payload/payloads/dnf/packages/packages.py

```diff
@@ -355,6 +368,30 @@ def set_missing_ignored(self, missing_ignored):
         self.missing_ignored_changed.emit()
         log.debug("Ignore missing is set to %s.", missing_ignored)
 
+    @property
+    def broken_ignored(self):
+        """Ignore packages that have conflicts with other packages.
+
+        :rtype: bool
+        """
+        return self._broken_ignored
+
+    def set_broken_ignored(self, broken_ignored):
+        """Set if the packages that have conflicts with other packages should be ignored.
+
+        :param missing_ignored: True if broken packages should be ignored.
+        :type missing_ignored: bool
+        :raise: UnsupportedValueError if ignorebroken is disabled on this platform.
```

Comment:
I think that we say `product` instead of `platform`, when we are talking about Fedora and RHEL.

Reply:
Fixed.